src/widgets/statistical-context-criteria/containers/statistical-context-criteria.jsx

```diff
@@ -86,6 +86,7 @@ export const mapStateToProps = (
             ...storeToArray(state?.metadataByType?.series),
           ]
         : storeToArray(state?.metadataByType?.series),
+    required: state?.metadataByType?.series?.length > 0,
```

Comment:
`state?.metadataByType?.series` field is a object (key, value).
So this object.length is always undefined.

A simply fix can be
```js
required: Object.keys(state?.metadataByType ?? {}).length > 0,
```